src/metadata_export.py

```diff
@@ -287,7 +287,7 @@ def save_release(self, data_record, node_type, node_id, crdc_id):
         current_date = current_datetime()
         if not existed_crdc_record or existed_crdc_record.get(DATA_COMMON_NAME) != self.submission.get(DATA_COMMON_NAME) \
             or existed_crdc_record.get(NODE_ID) != node_id or existed_crdc_record.get(NODE_TYPE) != node_type:
-            if self.submission.get(SUBMISSION_INTENTION) == SUBMISSION_INTENTION_DELETE:
+            if self.submission.get(SUBMISSION_INTENTION == SUBMISSION_INTENTION_DELETE):
```

Comment:
This look wrong to me, the original line in 2.0.0 seems to be correct.

Reply:
Updated. Thanks!